src/dashboards/teachers/details.jsonnet

```diff
@@ -82,25 +82,27 @@ dashboard.new(
   ).addTarget(
     elasticsearch.target(
       datasource=common.datasources.lrs,
-      query='%(video_query)s AND verb.id:"%(verb_played)s"' % {
+      query='%(video_query)s AND verb.id:"%(verb_played)s" AND %(time)s:[0 TO %(view_count_threshold)s]' % {
```

Comment:
This query is repeated multiple times across various dashboards, it should be factorized in `teachers_common.queries.*`